content/rdi/quickstart/write-behind-guide.md

```diff
@@ -39,16 +39,21 @@ RDI write-behind currently supports these target data stores:
 | Redis Enterprise     |
 | SQL Server |
 
-
 ## Prerequisites
 
 The only prerequisite for running RDI write-behind is [Redis Gears Python](https://redis.com/modules/redis-gears/) >= 1.2.6 installed on the Redis Enterprise Cluster and enabled for the database you want to mirror to the downstream data store.
 For more information, see [Redis Gears installation]({{<relref "/rdi/installation/install-redis-gears">}}).
 
-## Preparing the write-behind pipeline
+## Preparing the write-behind & read-through pipelines
```

Comment:
should be only write-behind at this point, read-through should be introduced in the (very near) future.

Reply:
done